lib/transaction/transaction.js

```diff
@@ -205,6 +205,10 @@ Transaction.prototype._isInvalidSatoshis = function() {
 };
 
 Transaction.prototype._hasFeeError = function(opts) {
+  if (this._getUnspentValue() < 0) {
```

Comment:
I'm not sure what you are suggesting. This check here looks very simple to me.
Is _getUnspentValue() expensive that we shouldn't call it more than once?